content/en/tracing/other_telemetry/connect_logs_and_traces/python.md

```diff
@@ -78,6 +79,10 @@ def hello():
 hello()
 ```
 
+#### Step 3 - Configure log collection
+
+Ensure that log collection is configured in the Datadog Agent and that the [Logs Agent configuration][2] for the specified files to tail is set to `source: python` so log pipelines can parse the log files. If the `source` is set to a value other than `python`, you may need to add a [trace remapper][3] to the appropriate log processing pipeline for the correlation to work correctly.
```

Comment:
For the part that said "Ensure that log collection is configured in the Datadog Agent" what specifically does the user need to do? I removed that part of the sentence because it seems to be the same as the second part of the sentence.

```suggestion
Ensure that files being tailed by the [Logs Agent configuration][2] are set to `source: python` so log pipelines can parse those files. If `source` is set to a value other than `python`, you may need to add a [trace remapper][3] to the appropriate log processing pipeline for the correlation to work correctly.
```